coderadar-ui/src/app/app.module.ts

```diff
@@ -62,17 +62,22 @@ import {PinchZoomModule} from 'ngx-pinch-zoom';
 import {ScrollingModule} from '@angular/cdk/scrolling';
 import { ListViewComponent } from './view/project-dashboard/list-view/list-view.component';
 import { BranchViewComponent } from './view/project-dashboard/branch-view/branch-view.component';
-import {ContributorCardComponent, ContributorDialogComponent} from './components/contributor-card/contributor-card.component';
+import {ContributorCardComponent} from './components/contributor-card/contributor-card.component';
 import { FileViewComponent } from './view/file-view/file-view.component';
-import {ContributorMergeDialogComponent} from './components/contributor-merge-dialog/merge-dialog.component';
-import {
-  DeleteProjectDialogComponent
-} from './components/delete-project-dialog/delete-project-dialog.component';
+import { TeamsDashboardComponent } from './view/teams-dashboard/teams-dashboard.component';
+import { AddTeamComponent } from './view/add-team/add-team.component';
+import { SidenavContentComponent } from './view/sidenav-content/sidenav-content.component';
+import {ContributorMergeDialogComponent} from './components/contributor-merge-dialog/contributor-merge-dialog.component';
+import {DeleteProjectDialogComponent} from './components/delete-project-dialog/delete-project-dialog.component';
+import {AddProjectToTeamDialogComponent} from './components/add-project-to-team-dialog/add-project-to-team-dialog.component';
+import {ContributorDialogComponent} from "./components/contributor-card/contributor-dialog.component";
```

Comment:
Please execute TSLint. It should emphasize doulbe quotes in ts files and more. TSLint issues should be fixed.

Reply:
Ts lint returns no errors for me. In fact running tslint --fix will remove all double quotes as those are counted as errors.

Reply:
I've run the linter again

Reply:
Thanks, I think it would be a good idea to make it part of the build process!